### DISCUSSION TITLE
SERVER TEMPORARILY NOT AVAILABLE

### DISCUSSION THREAD

**Discussion by Itseric100** [postalserver/postal#2008] [Help with using Postal]
Good day house, everything on my postal was working pretty well until recently, i have been able to send just about 7000 emails with my smtp and now i am getting a soft fail with 421. 4.1.0 (server temporarily not available) error, i have tried everything i could to make it work including reinstalling postal on my server, please is anyone else experiencing this as well or knows the way out, thanks in advance.

**Comment by willpower232** [comment]
Can you share a screenshot of the error message you're seeing? It might be that your server hosting company or whoever you're emailing has started to block you. You could also check block lists at mxtoolbox.com

**Comment by Itseric100** [comment]
i checked and my ip isnt blacklisted, if there is possibility that the company i am emailing has started to block me, will it accept new ip address in the ip pool?? does that work that way?? thanks so much for your time.  ![Screenshot 2022-06-03 061712](https://user-images.githubusercontent.com/96216385/171836782-f8ead295-1688-4ac9-b4c1-fe361338dbc4.png)   this is the error message i have been receiving.

**Reply by willpower232** [reply]
The email address in that error message is one for Postals return path so either your Postal is trying to email itself and has gotten confused or whoever you're emailing is validating the return path email address and can't connect to your Postal server.

**Reply by Itseric100** [reply]
oh oh, i am not a guru in this aspect, how can i fix it please so that everything can work properly and efficiently?? thanks so much for your time and assistance.

**Comment by Itseric100** [comment]
if any of them has started to block me, how can i fix it?
